clang/include/clang/Sema/CXXFieldCollector.h

```diff
@@ -65,7 +65,7 @@ class CXXFieldCollector {
 
   /// getCurFields - Pointer to array of fields added to the currently parsed
   /// class.
-  FieldDecl **getCurFields() { return &*(Fields.end() - getCurNumFields()); }
```

Comment:
I think this is basically [cwg232](https://wg21.link/cwg232) this is ok b/c we are not doing an lvalue-to-rvalue conversion but only using it to take the address of. 

CC @zygoloid

Reply:
Thanks for the review and linked pointers!
This seems to be the case. and generated code does only the pointer arithmetic, no actual load. Closing this as this is not an issue.

btw, I like your C++ trivia on twitter 😊